clang/include/clang/Serialization/ASTWriter.h

```diff
@@ -224,6 +224,7 @@ class ASTWriter : public ASTDeserializationListener,
   /// discovery) and start at 2. 1 is reserved for the translation
   /// unit, while 0 is reserved for NULL.
   llvm::DenseMap<const Decl *, LocalDeclID> DeclIDs;
+  // TMP: ^ DeclIDs type for reference
```

Comment:
I will remove all TMP comments before committing. Using this to simplify the review by pulling in the pointers to the relevant code.